src/coord/src/catalog/builtin.rs

```diff
@@ -1541,7 +1541,40 @@ GROUP BY operator, worker",
     needs_logs: true,
 };
 
-// Next id BuiltinView: 5034
+pub const PG_CONSTRAINT: BuiltinView = BuiltinView {
+    name: "pg_constraint",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_constraint AS SELECT
+    NULL::pg_catalog.oid as oid,
+    NULL::pg_catalog.varchar(64) as conname,
+    NULL::pg_catalog.oid as connamespace,
+    NULL::pg_catalog.char as contype,
+    NULL::pg_catalog.bool as condeferrable,
+    NULL::pg_catalog.bool as condeferred,
+    NULL::pg_catalog.bool as convalidated,
+    NULL::pg_catalog.oid as conrelid,
+    NULL::pg_catalog.oid as contypid,
+    NULL::pg_catalog.oid as conindid,
+    NULL::pg_catalog.oid as conparentid,
+    NULL::pg_catalog.oid as confrelid,
+    NULL::pg_catalog.char as confupdtype,
+    NULL::pg_catalog.char as confdeltype,
+    NULL::pg_catalog.char as confmatchtype,
+    NULL::pg_catalog.bool as conislocal,
+    NULL::pg_catalog.int4 as coninhcount,
+    NULL::pg_catalog.bool as connoinherit,
+    NULL::pg_catalog.int2[] as conkey,
+    NULL::pg_catalog.int2[] as confkey,
+    NULL::pg_catalog.oid[] as conpfeqop,
+    NULL::pg_catalog.oid[] as conppeqop,
+    NULL::pg_catalog.oid[] as conffeqop,
+    NULL::pg_catalog.oid[] as conexclop
+    WHERE false",
```

Comment:
If you're going to add all the columns, might as well add conbin here as a text field. Or only add columns needed by metabase. Dunno which is better.

Reply:
I didn't add it because I didn't know what was the best way to represent `pg_node_tree` internally: if string is good enough, I'll add it.